include/slang/util/FlatHashMap.h

```diff
@@ -0,0 +1,5 @@
+#include <flat_hash_map.hpp>
+
+#include "slang/util/Hash.h"
+template<typename K, typename V, typename H = slang::Hasher<K>, typename E = std::equal_to<K>,
```

Comment:
You can just put this in Hash.h, no need for a separate file for just one definition. Also it should be namespaced anyway.

Reply:
Also I think this file is causing the build to fail due to lack of #pragma once